src/database/db_connection.rs

```diff
@@ -267,6 +267,11 @@ impl DbBackend {
             Self::Sqlite => Box::new(SqliteQueryBuilder),
         }
     }
+
+    /// Check if the database supports `RETURNING` syntax on insert and update
+    pub fn support_returning(&self) -> bool {
+        matches!(self, Self::Postgres)
```

Comment:
If self is a `Mock`, we should respect the `get_database_backend` of it.

Reply:
> If self is a Mock, we should respect the get_database_backend of it.

Yes! We should

> I still don't understand why the behaviour of insert and update differs though.

Does my answer clear enough?